jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -10,13 +10,78 @@
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import (PluginTemplateMixin, ViewerSelect, LayerSelect,
                                         PlotOptionsSyncState)
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.core.tools import ICON_DIR
 
 __all__ = ['PlotOptions']
 
 
 @tray_registry('g-plot-options', label="Plot Options")
 class PlotOptions(PluginTemplateMixin):
+    """
+    The Plot Options Plugin gives access to per-viewer and per-layer options and enables
+    setting across multiple viewers/layers simultaneously.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``multiselect``:
+      whether ``viewer`` and ``layer`` should both be in multiselect mode.
+    * ``viewer`` (:class:`~jdaviz.core.template_mixin.ViewerSelect`):
+    * ``layer`` (:class:`~jdaviz.core.template_mixin.LayerSelect`):
+    * :meth:`select_all`
+    * ``layer_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``subset_visible`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``show_axes`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`)
+    * ``collapse_function`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      only avilable for Cubeviz
+    * ``line_color`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``line_width`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``line_opacity`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``as_steps`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``uncertainty`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Imviz
+    * ``stretch`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``stretch_perc`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
+      not available for Specviz
+    * ``stretch_min`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):

```

Comment:
```suggestion
    * ``v_min`` (:class:`~jdaviz.core.template_mixin.PlotOptionsSyncState`):
```

Reply:
Re: https://github.com/spacetelescope/jdaviz/pull/1637#discussion_r973033131 -- "v_min" is familiar to those who already use matplotlib (and also what the API actually sets under the hood if you look at how astrowidgets API implemented "cuts".

I don't think it actually sets the stretch? 🤷

Reply:
It's coupled directly with the (stretch) percentile.  When you set the percentile to 95%, the min/max set automatically, and when you override the min/max, the percentile changes to "Custom".  So I think a common prefix of some sort to tie these together would be useful for the user.

Reply:
percentile is changed via the "cuts" API in astrowidgets, not "stretch".

Reply:
ok, then it looks like we need to dig into the code and try to resolve these discrepancies (between glue behavior based on the state objects and the astrowidgets API).  As far as I know, this does not interact with astrowidgets API at all - unless there is some callback under-the-hood that I haven't seen.

Reply:
glue isn't aware of astrowidgets, and astrowidgets was designed without glue input. But since we have astrowidgets hook-up now, I think it is less confusing for users if they can vaguely map this onto the public astrowidgets API.

Reply:
The glue state objects are also called `v_min/max` and are (luckily) what astrowidgets API reads/sets under-the-hood.  How about `stretch_vmin/max`?  (But also will defer to @Jenneh on how to prioritize prefixes, staying consistent with glue state names, staying consistent with astrowidgets API, and grouping in the API listing will ultimately be most convenient/intuitive for the user)